pyproject.toml

```diff
@@ -81,6 +81,7 @@ dependencies = [
     "curies>=0.10.6",
     "python-dateutil",
     "networkx>=3.4",
+    "httpx",
```

Comment:
Because httpx has been configured to use HTTP2, as in

``` python
with Client(http2=True, timeout=Timeout(timeout)) as client:
```

, the following change should be applied to ensure that [HTTP2 support is enabled](https://www.python-httpx.org/http2/#enabling-http2):

``` suggestion
    "httpx[http2]",
```

Reply:
Thanks! I missed that locally as well, so should've remembered it.